.github/workflows/flaky-test-monitor.yml

```diff
@@ -35,7 +35,6 @@ jobs:
       COMMIT_SHA: ${{ github.sha }} 
       RUN_ID: ${{ github.run_id }}
       RESULTS_FILE: test-results
-      TEST_OUTPUT_FILE: test-output
```

Comment:
any reason we removed this?

Reply:
The `run-tests.sh` script no longer writes to a file, it just prints to stdout. This way it can be used for both CI and flaky test monitor. For flaky test monitor, we just need to then pipe the output to a file.